### PR TITLE
Logic: add opencom command

### PR DESCRIPTION
**Pull request by seelengxd**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/OpenCommissionCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.OpenCommissionCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new OpenCommissionCommand object
```
Comment (chloeelim) [inline src/main/java/seedu/address/logic/parser/OpenCommissionCommandParser.java:10]: Missing punctuation at the end of the Javadoc header?

File: src/main/java/seedu/address/logic/parser/OpenCommissionCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+public class OpenCommissionCommandParser implements Parser<OpenCommissionCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the OpenCommissionCommand
+     * and returns a OpenCommissionCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```
Comment (chloeelim) [inline src/main/java/seedu/address/logic/parser/OpenCommissionCommandParser.java:17]: Missing punctuation at the end of the JavaDoc here as well I believe?

File: src/main/java/seedu/address/logic/commands/OpenCommissionCommand.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import seedu.address.model.commission.Commission;
+
+/**
+ * Selects a commission and shows their details.
+ */
+public class OpenCommissionCommand extends Command {
+    public static final String COMMAND_WORD = "opencom";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens a commission and shows commission details"
```
Comment (chloeelim) [inline src/main/java/seedu/address/logic/commands/OpenCommissionCommand.java:20]: The phrasing here is a little weird, perhaps we should try to make this clearer?

**Review by chloeelim — CHANGES_REQUESTED** [review]
LGTM (almost), just some very minor styling issues to address.

**Review by zlimez — APPROVED** [review]
LGTM

**Review by chloeelim — APPROVED** [review]

**Pull request merged**